src/components/HeaderWithLanguage/HeaderStatusInformation.jsx

```diff
@@ -14,6 +14,7 @@ import { Check, AlertCircle } from '@ndla/icons/editor';
 import Tooltip from '@ndla/tooltip';
 import HowToHelper from '../HowTo/HowToHelper';
```

Comment:
Denne må nok fjernes for at linteren skal være fornøgd.

Reply:
hadde glemt format også så var ikke bare-bare😅